data/amd/register/cpuid/CPUID_Fn80000019_EAX.yml

```diff
@@ -0,0 +1,64 @@
+- name: CPUID_Fn80000019_EAX
```

Comment:
It looks like you missed a file CPUID_Fn80000019_E**B**X, which should come right after this file